### PR TITLE
Put in the field wire

### PR DESCRIPTION
**Pull request by myliu-hub**

### CODE REVIEW

**Comment by mirguest** [comment]
Hi @myliu-hub ,
Do you have any reference about your change? What are the placements of the field wires?
Tao

**Comment by myliu-hub** [comment]
> Hi @myliu-hub ,
> Do you have any reference about your change? What are the placements of the field wires?
> Tao

In each cell, there is one signal wire and eight field wires, which can be shared with surrounding cells and  used to provide an electric field.
The following figure shows the distribution of the drawn wires, the red is the signal wire, and the green is the field wire:

**Comment by myliu-hub** [comment]
4985b3001de63f77a5ef59644480f9d

**Comment by mirguest** [comment]
Hi @myliu-hub,
For the cells in the same layer, I can understand how the field wires are shared. But for the different layers, how can you make sure they are shared? 
What does your figure mean?
Tao

**Comment by myliu-hub** [comment]
> Hi @myliu-hub,
> For the cells in the same layer, I can understand how the field wires are shared. But for the different layers, how can you make sure they are shared?
> What does your figure mean?
> Tao
I understand the cell as a square, so I think that different layers can be shared.

**Comment by myliu-hub** [comment]
Sorry, I carefully studied the ratio of signal wire to field wire should be 1:5, but it was only 1:3.

**Comment by myliu-hub** [comment]
The material of the inserted inner and outer walls is currently set to air, and the inner diameter of the inner cylinder is reduced to 234mm, and the outer diameter of the outer cylinder is increased to 1720mm.

File: Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp [hunk 1]
```diff
@@ -16,6 +16,9 @@
 #include "DDSegmentation/Segmentation.h"
 #include "DetSegmentation/GridDriftChamber.h"
 
+#include "TFile.h"
+#include "TTree.h"
+
```
Comment (mirguest) [inline Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp:21]: Why add TFile and TTree?

File: Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp [hunk 2]
```diff
@@ -92,9 +95,31 @@ static dd4hep::Ref_t create_detector(dd4hep::Detector& theDetector,
     dd4hep::Tube det_outer_chamber_solid(outer_chamber_radius_min, outer_chamber_radius_max, outer_chamber_length*0.5);
     dd4hep::Volume det_outer_chamber_vol(det_name+"_outer_chamber_vol", det_outer_chamber_solid, det_mat);
 
+    // - inner wall
+    xml_coll_t c(x_det,_U(barrel));
```
Comment (mirguest) [inline Detector/DetDriftChamber/src/driftchamber/DriftChamber.cpp:99]: I think it is not a good idea to use the "barrel" and "model" here.

**Review by mirguest — APPROVED** [review]
Looks good. Thanks!
Tao

**Pull request merged**